packages/core/src/civil.ts

```diff
@@ -130,6 +119,17 @@ export class Civil {
     return artifacts.OwnedAddressTCRWithAppeals.networks[networkId].address;
   }
 
+  /**
+   * Returns a OwnedAddressTCRWithAppeals object, which is an abstraction layer to
+   * the smart-contract located on Ethereum in the current network. Instance returned
+   * is associated with the contract deployed via truffle migrations (address is
```

Comment:
Knowledge of the address being in some internal arttifacts is not helpful to the library *user*